### ISSUE TITLE
Limit records are shown in graph view when using group by

### ISSUE THREAD

**Issue by QuangTrieu2919** [odoo/odoo#113481]
Hi, I want to limit the number of char have been shown in graph view (bar) when I using group by.
Is it possible and how can I handle it?

**Comment by AbdelAzizMohamedMousa** [comment]
Yes, it is possible to limit the number of records shown in the graph view when using group by in Odoo.

You can achieve this by setting a limit on the domain of the graph view. Here's an example of how to do this:

Go to the view for which you want to set the limit (e.g. Sales > Sales Orders > Graph View).
Click on the "Edit" button to open the view in edit mode.
In the "Graph" tab, scroll down to the "Advanced Search" section.
In the "Domain" field, add the following code snippet:
[('your_group_by_field', '!=', False)] + (your_domain_expression_here)[:-1] + (your_limit_expression_here)

Replace "your_group_by_field" with the name of the field you are grouping by, "your_domain_expression_here" with the domain expression you are using, and "your_limit_expression_here" with the limit expression.

For example, if you are grouping by the "partner_id" field and want to limit the graph to the top 10 partners by total amount, you could use the following code snippet:

[('partner_id', '!=', False)] + [('state', '=', 'sale')][:-1] + [('partner_id', 'in', top_10_partner_ids)]

Where "top_10_partner_ids" is a list of the IDs of the top 10 partners you want to show in the graph.

Click on the "Save" button to save the changes.
After setting the domain with the limit expression, the graph view will only show the number of records you specified in the expression.
